pkg/networking/logging.go

```diff
@@ -14,7 +15,21 @@ import (
 const defaultNetworkLogLevel = zerolog.DebugLevel
 const extendedNetworkLogLevel = zerolog.TraceLevel
 const maxNumberOfRequestBodyCharacters = 60
-const maxNumberOfResponseBodyCharacters = -1 // log complete response body
+const maxNumberOfResponseBodyCharacters = 10 * 1024 // 10KiB, to ensure we don't log an entire CLI download response that is interrupted.
```

Comment:
Praise: I think setting `maxNumberOfResponseBodyCharacters`  is the best minimal solution.